train.py

```diff
@@ -151,6 +155,7 @@ def create_datasets(tokenizer, args):
         dataset = dataset.train_test_split(test_size=0.005, seed=args.seed)
         train_data = dataset["train"]
         valid_data = dataset["test"]
+        train_data = train_data.shuffle(seed=args.seed)
```

Comment:
we probably don't need this as the data is already shuffled with `train_test_split`?